api/AudioParam.json

```diff
@@ -288,7 +288,9 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "25"
+              "version_added": "25",
+              "partial_implementation": true,
+              "notes": "Does not work (see <a href='https://bugzil.la/1171438'>bug 1171438</a>)."
```

Comment:
One minor suggestion: since the second bug (https://bugzilla.mozilla.org/show_bug.cgi?id=1567777 the one closed as a duplicate) speaks directly to the issue with this method, I'd suggest linking to it as well (also for the other method below).

Reply:
Hi @s0rent, do you plan to revisit this PR soon?